Challo/Challo/Modules/Dashboard/DashboardViewComponents/PastBookingCard.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  PastBookingCard.swift
+//  Challo
+//
+//  Created by Tan Le Yang on 12/4/21.
+//
+
+import SwiftUI
+
+struct PastBookingCard: View {
```

Comment:
I think can just let `BookingCard` accept a `ViewBuilder` in order to insert more action icons at the topright